backend/environment.ts

```diff
@@ -1,2 +1,2 @@
-export const databaseName = 'mongodb://localhost/recipebook';
+export const databaseName = 'mongodb://192.168.56.104:27017/recipebook';
```

Comment:
Need this to be changed back to localhost

Reply:
Hot fix is in the pulls.